src/clients/ethereum.ts

```diff
@@ -22,9 +22,16 @@ export type EthCall = {
   callInput?: string,
 }
 
+type returnType = ethers.utils.LogDescription & {
```

Comment:
hm what's this used for?

Reply:
it's the return type for getEventsFrom. I renamed it to Events